dm/worker/subtask.go

```diff
@@ -86,12 +86,12 @@ type SubTask struct {
 }
 
 // NewSubTask creates a new SubTask
-func NewSubTask(cfg *config.SubTaskConfig) *SubTask {
-	return NewSubTaskWithStage(cfg, pb.Stage_New)
+func NewSubTask(cfg *config.SubTaskConfig, enableRelay bool) *SubTask {
```

Comment:
How about removing this `enableRelay` too?

Reply:
update in 1c061d8